wpiutil/src/main/native/windows/StackWalker.cpp

```diff
@@ -484,8 +485,8 @@ class StackWalkerInternal
       if (hToolhelp == NULL)
         continue;
       createToolhelp32Snapshot = (tCT32S)GetProcAddress(hToolhelp, "CreateToolhelp32Snapshot");
-      module32First = (tM32F)GetProcAddress(hToolhelp, strModule32First);  
-      module32Next = (tM32N)GetProcAddress(hToolhelp, strModule32Next); 
+      module32First = (tM32F)GetProcAddress(hToolhelp, strModule32First);
+      module32Next = (tM32N)GetProcAddress(hToolhelp, strModule32Next);
```

Comment:
These changes seem unrelated.

Reply:
This was just because my editor is set to always remove trailing white space, which this file must have had. I had to edit this file because it was getting an odd linker error, likely an ordering issue, so I had to add a new import.